todo_app/app.py

```diff
@@ -1,51 +1,75 @@
-from flask import Flask, render_template, request, redirect
-
-from todo_app.data.session_items import (
-    get_items,
-    add_item,
-    remove_item,
-    get_item,
-    save_item,
-    Status,
-)
+from flask import Flask, redirect, render_template, request
+
+from todo_app.data.trello_items import Status, TrelloRequests
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
+trello_requests = TrelloRequests()
+
+
 @app.route("/")
 def index():
-    completed_items = []
-    incomplete_items = []
-    for item in sorted(get_items(), key=lambda item: item["id"]):
-        if item["status"] == Status.COMPLETED:
-            completed_items.append(item)
-        else:
-            incomplete_items.append(item)
+    trello_requests.init_list_maps()
```

Comment:
I'm always a bit wary of initialisation steps that need to be explicitly called by dependent code - it's easy for this to be missed and then pass because it happens to usually occur elsewhere only for you to later discover that's not a guarantee. Baking this into other methods is usually safer - is there any reason not to just run this immediately during the trello_requests `init` function?

Reply:
My thoughts were that if it happened in `init`, and this then gives an error, handling it is more awkward. E.g we can't go to the error page.